core/src/test/scala/org/typelevel/discipline/laws.scala

```diff
@@ -23,13 +23,14 @@ package org.typelevel.discipline
 
 import org.scalacheck.Prop
 
-object Dummy {
+import scala.language.adhocExtensions
```

Comment:
Why exactly is this import necessary?

Reply:
Good point, actually I don't think it is. Just have to mark the classes `open`.

Reply:
```scala
[warn] -- Feature Warning: /workspace/discipline-specs2/core/src/test/scala/org/typelevel/discipline/laws.scala:70:12 
[warn] 70 |  ) extends DefaultRuleSet(name, parent, props*)
[warn]    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[warn]    |Unless class DefaultRuleSet is declared 'open', its extension in a separate file should be enabled
```

Reply:
Actually we do need it then I think.

Reply:
`open` doesn't work?

Reply:
`DefaultRuleSet` is not defined here so we can't change it.

Reply:
That comes from `specs2`?

Reply:
No, it comes from discipline.
https://github.com/typelevel/discipline/blob/79a121e3e0c605bcd30f73fa3f88830b521e35c4/core/src/main/scala/org/typelevel/discipline/Laws.scala#L107

Reply:
Cool. Thanks.